src/shared/income-filter/test/income-filter.component.spec.ts

```diff
@@ -1,60 +1,133 @@
-import { ComponentFixture, TestBed, async } from '@angular/core/testing';
-import { TranslateModule, TranslateService, TranslateLoader, TranslateParser } from 'ng2-translate';
-import { StoreModule } from '@ngrx/store';
-import {
-    MathService
-} from '../../../common';
-import {
-    TranslateLoaderMock,
-    TranslateParserMock,
-    TranslateServiceMock
-} from '../../../test/';
-import { StreetFilterComponent } from '../../street-filter/street-filter.component';
-import { StreetFilterDrawService } from '../../street-filter/street-filter.service';
-import { IncomeFilterComponent } from '../income-filter.component';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {NO_ERRORS_SCHEMA} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {Http} from '@angular/http';
+import {Store} from '@ngrx/store';
+import {Observable} from 'rxjs/Observable';
+
+import {TranslateLoader, TranslateModule, TranslateStaticLoader} from 'ng2-translate';
+
+import {MathService} from '../../../common';
+import {StreetFilterDrawService} from '../../street-filter/street-filter.service';
+import {IncomeFilterComponent} from '../income-filter.component';
+
+const defaultStreetSettings = {
+  lowIncome: 30,
+  highIncome: 60
```

Comment:
move this to the bottom of the page

Reply:
it's impossible to use const before it's declaration

Reply:
no, sorry. Variables declared outside of Describes will be initialized before. So it's fine in this case. Done.